changelog/fragments/image-digests.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      Adds support to bundle operators using image digests instead of tags.
+    kind: "addition"
+    breaking: false
```

Comment:
We are changing the scaffold. So, we need to provide the migration steps for authors to know how they need to update their makefile to have it.

Reply:
Done https://github.com/operator-framework/operator-sdk/pull/5567/commits/80d7022de4de649c713eb18f7a549ee3009d8375